### ISSUE TITLE
BrowserWindow background color is white after reload

### ISSUE THREAD

**Issue by adill** [electron/electron#14976]
* Output of `node_modules/.bin/electron --version`: v3.0.2
* Operating System (Platform and Version): Windows 10 Pro 17134
* Output of `node_modules/.bin/electron --version` on last known working Electron version (if applicable): v1.6.15

**Expected Behavior**
Reloading (Ctrl R) a `BrowserWindow` with `backgroundColor` set should display that background color until the page begins to load.

**Actual behavior**
Reloading (Ctrl R) a `BrowserWindow` with `backgroundColor` set displays white until the page begins to load.

**To Reproduce**
I've reproduced the issue in electron-quick-start by:
* Adding ` style="background-color: transparent;"` to `index.html`'s `body` tag (pedantic, I know)
* Adding `backgroundColor: '#ffff0000', ` to `main.js`'s `BrowserWindow` options
* Alternatively, clone my fork of electron-quick-start:
```sh
$ git clone https://github.com/adill/electron-quick-start.git -b background-color-reload
$ npm install
$ npm start
```
1. Notice the red background color on the page (transparent body is showing through to the opaque red background.)
2. Reload (Ctrl R)
3. Notice the background color is now white

**Comment by codebytere** [comment]
Duplicate of https://github.com/electron/electron/issues/12211

**Comment by adill** [comment]
@codebytere I'm not sure that issue is the same as this one. The linked issue appears to describe un-desired state sharing related to the experimental affinity options.

I've fixed my issue here: https://github.com/discordapp/electron/tree/fix-bgcolor-reload

On reload `RenderViewCreated` appears to fire before the `WebFrameWidget` has been attached (it is `nullptr`), however, `RenderFrameCreated` is fired **after** we've created and attached the render view.

I can submit a PR once I've tackled the new GN-based build process for 4.x.
